tests/e2e/tests/basic.ts

```diff
@@ -0,0 +1,62 @@
+import { Selector } from 'testcafe';
+import { readFileSync } from 'fs';
+import * as path from 'path';
+import * as got from 'got';
+
+
+const maxAppWarmup = 15;
+const rootURL = process.env.SIGNHASH_URL || 'http://localhost:8080';
+
+
+const knownFile = {
+  path: 'foo.txt',
+  name: 'foo.txt',
+  hash: readFileSync(path.join(__dirname, 'foo.hash')).toString().trim()
+};
+
+
+fixture`SignHash`
+  .page(rootURL)
+  .before(waitForPage(maxAppWarmup));
+
+
+test('Site is available', async t => {
+  await t
+    .expect(Selector('body').textContent)
+    .contains('SignHash');
+});
+
+
+test('Verifying known hash', async t => {
+  const file = knownFile;
+  const text = Selector('body').textContent
```

Comment:
Missing semicolon at the end of the line

Reply:
Weird, that my linter didn't caught that.